blockchain/storagev2/leveldb/leveldb_test.go

```diff
@@ -0,0 +1,256 @@
+package leveldb
+
+import (
+	"context"
+	"crypto/rand"
+	"math/big"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/blockchain"
+	"github.com/0xPolygon/polygon-edge/blockchain/storagev2"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+	"github.com/stretchr/testify/require"
+)
+
+func newStorage(t *testing.T) (*storagev2.Storage, func()) {
```

Comment:
Moved to testing.go. The only problem is Uncles since it requires import of blockchain package into storagev2, but storagev2 must be imported into blockchain and this makes circular reference. I will leave it so far without Uncles set, it is set to blockchain.NewTestHeaders(10)